cmd/controller-manager/main.go

```diff
@@ -90,7 +92,12 @@ func main() {
 		klog.Info("HELM_RELEASE environment variable not set")
 	}
 
-	cfg, err := rest.InClusterConfig()
+	kubconfig := os.Getenv("KUBE_CONFIG")
```

Comment:
@zhucan Please use the `KUBECONFIG` env as kubectl uses.

Reply:
done